test/trivial/diten/testIntExp4.good

```diff
@@ -1 +1 @@
-error: 0 cannot be raised to a negative power
```

Comment:
For type-tests.isSubtype.good, the compiler is calling USR_WARN with ast==NULL for some reason.